packages/commons-test/.env.test

```diff
@@ -1,6 +1,7 @@
 LOG_LEVEL=info
 S3_BUCKET=interop-local-bucket
 AWS_CONFIG_FILE=aws.config.local
+AWS_SES_ENDPOINT=http://localhost:8021
```

Comment:
This is probably not needed. We don't use the AWS SES container in the tests written in commons-test